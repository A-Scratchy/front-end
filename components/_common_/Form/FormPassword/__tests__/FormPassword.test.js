/* eslint-env jest */
import React from 'react';
import createSnapshotTest from 'test-utils/createSnapshotTest';

import FormPassword from '../FormPassword';

describe('FormPassword', () => {
  it('should render with required props', () => {
    createSnapshotTest(<FormPassword id="test" />);
  });
});
