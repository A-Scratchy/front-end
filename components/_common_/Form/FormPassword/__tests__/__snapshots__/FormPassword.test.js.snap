// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormPassword should render with required props 1`] = `
<div
  className="formInput"
>
  <input
    id="test"
    onChange={[Function]}
    placeholder={null}
    type="password"
    value=""
  />
</div>
`;
